tests/network-ovn

```diff
@@ -99,7 +100,7 @@ lxc config set storage.images_volume=default/images # Speeds up instance creatio
 setupCloudInitReady default
 
 setupInstanceImage() {
-  lxc image copy ubuntu-minimal:22.04 local: --alias jammy-minimal
+  lxc image copy ubuntu:22.04 local: --alias jammy-minimal
```

Comment:
Shall we call the image alias just jammy?

Reply:
You don't really need this line, can just create the temporary instance directly from the remote image before publishing as jammy-small.